lib/audit/catalog_to_moab.rb

```diff
@@ -142,4 +143,23 @@ def online_moab_found?(druid, storage_dir)
     false
   end
 
+  # given whether the caller found the expected version of preserved_copy on disk, this will perform
+  # other validations of what's on disk, and will update the status accordingly.
+  def set_status_as_seen_on_disk(found_expected_version)
+    unless moab_validation_errors.empty?
+      update_status(PreservedCopy::INVALID_MOAB_STATUS)
+      return
+    end
+
+    unless found_expected_version
+      update_status(PreservedCopy::EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS)
+      return
+    end
+
```

Comment:
Just noticing this common pattern which might or might not suggest refactoring : we have several places where we `do x and return unless something`

Reply:
good observation, @tallenaz .  I feel it is a bit too early to refactor;  I also figure it's a common thing more broadly than this code base (like, out in ruby land)